Week 2/shopee/src/components/Cart.jsx

```diff
@@ -0,0 +1,87 @@
+              </g>
+            </svg>
+            <h4 className="">| Giỏ Hàng</h4>
+          </div>
+        </div>
+        {cart.length > 0 ? (
+          <div className="cart__bg">
+            <div className="cart__content">
+              <div className="content__title">
+                <span>Tên sản phẩm</span>
+                <span>Đơn giá</span>
+                <span>Số lượng</span>
+                <span>Thành tiền</span>
+                <span>Thao tác</span>
+              </div>
+              {cart.map((product) => (
+                <div className="content__text" key={product.id}>
+                  <span>{product.title}</span>
+                  <span>{product.price}</span>
+                  <input
+                    min={1}
+                    type="number"
+                    value={product.quantity}
+                    onChange={(e) =>
+                      dispatch({
+                        type: "CHANGE_QUANTITY",
+                        payload: {
+                          id: product.id,
+                          quantity: e.target.value,
+                        },
+                      })
+                    }
```

Comment:
@phamhuutinh0605 : không được để body function trực tiếp trên component move riêng ra, những chổ khác cũng vậy
```
const handleOnChangeQuantity = (e) =>
                      dispatch({
                        type: "CHANGE_QUANTITY",
                        payload: {
                          id: product.id,
                          quantity: e.target.value,
                        },
                      })
```
```suggestion
                    onChange={handleOnChangeQuantity}
```